### ISSUE TITLE
Add docs on how to use @nrwl/workspace builder

### ISSUE THREAD

**Issue by iangregsondev** [nrwl/nx#2060]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x ] I checked the documentation and found no answer
- [x ] I checked to make sure that this issue has not already been filed
- [x ] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

## Expected Behavior

Need to know how to use the run-commands, currently i get an error

## Current Behavior

i run 

```
ng run build my-test
```
 and have this

```
      "architect": {
        "build": {
          "builder": "@nrwl/workspace:run-commands",
          "readyWhen": "I have finsihed",
          "commands": [
            {
              "command": "ls -a"
            },
            {
              "command": "ls -l"
            }
          ],
```

but I get the following error

```
Schema validation failed with the following errors:
  Data path "" should have required property 'commands'.
```

It also states in the docs that you can pass arguments and then get them using args array. Do you have a sample ?

I would like to use the above to host an application actually in the workspace that is not supported by nx 

Basically a react native application, and i would be calling  the EXPO start commnad :-)

Any help you can offer would be really appreciated

**Comment by iangregsondev** [comment]
Because react native is technically a react application, i was going to use the schematics etc for react but of course, I need a custom build / serve command - so the above builder sounds just like what i need.

But I don't have a solid example to work with so maybe i am doing it wrong and that's why I get an error

**Comment by isaacplmann** [comment]
You're missing an `options` key in your builder definition.

```
"compodoc": {
  "builder": "@nrwl/workspace:run-commands",
  "options": {
    "commands": [
      {
        "command": "npx compodoc -p apps/my-app/tsconfig.app.json"
      }
    ]
  }
},
```

See [this recipe] for a step by step walk through.

**Comment by Hotell** [comment]
this can be closed right ? cc @isaacplmann 

also API docs describe it quite well https://nx.dev/web/api/workspace/builders/run-commands

@iangregsondev 
> It also states in the docs that you can pass arguments and then get them using args array. Do you have a sample ?

Check following https://github.com/nrwl/nx/issues/2241#issuecomment-570328343

**Comment by iangregsondev** [comment]
Sure thank you. I will close it.
